src/main/kotlin/com/toasttab/pulseman/state/UserFeedback.kt

```diff
@@ -16,35 +16,32 @@
 package com.toasttab.pulseman.state
 
 import androidx.compose.runtime.Composable
-import androidx.compose.runtime.MutableState
-import androidx.compose.runtime.mutableStateOf
-import androidx.compose.ui.text.input.TextFieldValue
+import androidx.compose.runtime.mutableStateListOf
+import androidx.compose.runtime.snapshots.SnapshotStateList
 import com.toasttab.pulseman.view.userFeedbackUI
 import java.time.LocalDateTime
 import java.time.format.DateTimeFormatter
 
-class UserFeedback(private val userFeedback: MutableState<TextFieldValue> = mutableStateOf(TextFieldValue())) {
-
+class UserFeedback(
+    private val userFeedback: SnapshotStateList<String> = mutableStateListOf()
+) {
     companion object {
         private val formatter: DateTimeFormatter = DateTimeFormatter.ofPattern("HH:mm:ss")
     }
 
     fun setUserFeedback(text: String) {
-        val appendNewLine = if (userFeedback.value.text.isNotEmpty()) "\n" else ""
-        userFeedback.value = TextFieldValue(
-            text = "${userFeedback.value.text}$appendNewLine${timeNow()}: $text"
-        )
+        userFeedback.add("${timeNow()}: $text${System.lineSeparator()}")
```

Comment:
I added the `${System.lineSeparator()}` so that when you copy the lines, and paste it, it actually looks like different lines, and not just a single line with everything on it.